### PR TITLE
#13-open-firebase-repository

### PR DESCRIPTION
**Pull request by goranesine**

### CODE REVIEW

File: lib/ui_pages/home.dart [hunk 1]
```diff
@@ -2,18 +2,21 @@ import 'package:flutter/material.dart';
...
 import 'package:playground_app/widgets/drawer.dart';
 
-
 class HomePage extends StatelessWidget {
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      endDrawer: CustomDrawer(),
+      endDrawer: CustomDrawer(
```
Comment (matejthetree) [inline lib/ui_pages/home.dart]: fmt

File: lib/ui_pages/home.dart [hunk 2]
```diff
@@ -2,18 +2,21 @@ import 'package:flutter/material.dart';
...
+      ),
       appBar: AppBar(
         automaticallyImplyLeading: false,
-        title: Text('Mokuteki.io Playgrounds'),
+        title: Text(
+            'Mokuteki.io Playgrounds'),
         backgroundColor: Colors.pink,
```
Comment (matejthetree) [inline lib/ui_pages/home.dart]: remove

File: lib/ui_pages/home.dart [hunk 3]
```diff
@@ -2,18 +2,21 @@ import 'package:flutter/material.dart';
...
       ),
-      body: CoursesList(),
+      body: CoursesList(
```
Comment (matejthetree) [inline lib/ui_pages/home.dart]: fmt

File: lib/ui_pages/splash_screen.dart [hunk 4]
```diff
@@ -17,6 +18,14 @@ class _SplashScreenState extends State<SplashScreen>
     null,
   ];
 
+  Future<void> _signInAnonymously() async {
```
Comment (matejthetree) [inline lib/ui_pages/splash_screen.dart]: create FirebaseService

create this function inside firebase service
In providers map FirebaseService to value

File: lib/ui_pages/splash_screen.dart [hunk 5]
```diff
@@ -17,6 +18,14 @@ class _SplashScreenState extends State<SplashScreen>
...
+    try {
+      await FirebaseAuth.instance.signInAnonymously();
+    } catch (e) {
+      print(e);
```
Comment (matejthetree) [inline lib/ui_pages/splash_screen.dart]: add `throw e`

File: lib/ui_pages/splash_screen.dart [hunk 6]
```diff
@@ -31,6 +40,7 @@ class _SplashScreenState extends State<SplashScreen>
 
   @override
   void initState() {
+    _signInAnonymously();
```
Comment (matejthetree) [inline lib/ui_pages/splash_screen.dart]: here you will add Future() that will call 
`Provider.of<FirebaseService>(context).signInAnonymously()`

File: test/golden_test.dart [hunk 7]
```diff
@@ -1,7 +1,6 @@
 import 'package:flutter_test/flutter_test.dart';
 import 'package:flutter/material.dart';
 import 'package:playground_app/ui_pages/splash_screen.dart';
```
Comment (matejthetree) [inline test/golden_test.dart]: call golden test the same name as the file, and place in same directory `test/ui_pages/splash_screen_test.dart`

**Review by matejthetree — CHANGES_REQUESTED** [review]

File: lib/ui_pages/home.dart (hunk 1 above)
Comment (goranesine) [reply lib/ui_pages/home.dart]: Fixed

File: lib/ui_pages/splash_screen.dart (hunk 4 above)
Comment (goranesine) [reply lib/ui_pages/splash_screen.dart]: I move the function in firebase service.

File: lib/ui_pages/splash_screen.dart (hunk 5 above)
Comment (goranesine) [reply lib/ui_pages/splash_screen.dart]: moved from splshScreen

File: lib/ui_pages/splash_screen.dart (hunk 6 above)
Comment (goranesine) [reply lib/ui_pages/splash_screen.dart]: Fixed

File: test/golden_test.dart (hunk 7 above)
Comment (goranesine) [reply test/golden_test.dart]: Fixed

File: lib/ui_pages/splash_screen.dart (hunk 4 above)
Comment (goranesine) [reply lib/ui_pages/splash_screen.dart]: Map FirebaseService to value ?

**Pull request closed without merging**